src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/ManagedIndexRunner.kt

```diff
@@ -119,6 +121,7 @@ object ManagedIndexRunner : ScheduledJobRunner,
     private val errorNotificationRetryPolicy = BackoffPolicy.exponentialBackoff(TimeValue.timeValueMillis(250), 3)
     @Suppress("MagicNumber")
     private var jobInterval: Int = DEFAULT_JOB_INTERVAL
+    private var allowList: List<String> = emptyList()
```

Comment:
Defaulting to all actions in settings, but want to default to none in runner so that it goes from [none] -> [some] rather than [all] -> [some] just to be secure. Moved into variable though for both places.